src/common/variables.c

```diff
@@ -555,15 +608,32 @@ static char *_get_base_value(dt_variables_params_t *params, char **variable)
   else if(_has_prefix(variable, "SEQUENCE"))
   {
     uint8_t nb_digit = 4;
+    guint shift = 1;
+    // old $(SEQUENCEn) syntax for backwards compatibility
     if(g_ascii_isdigit(*variable[0]))
     {
       nb_digit = (uint8_t)*variable[0] & 0b1111;
       (*variable) ++;
     }
+    // new $(SEQUENCE[n,m]) syntax
+    // allows \[[0-9]+,[0-9]+] (PCER)
+    // everything else will be ignored
+    else if(*variable[0] == '[') 
+    {
+      char *parameters[2] = {NULL};
+      int num = _get_parameters(variable, parameters, 2);
```

Comment:
const

Reply:
what do you mean here? Replace 2 with a constant? Or make something a constant?

Reply:
Sorry, just style, make int num const.

`const int num = ...`